src/utils/generateTerms.ts

```diff
@@ -0,0 +1,12 @@
+// executed by node.js
+
+import strings from '../locales/strings'
+import { writeFileSync } from 'fs'
+const en = {}
+
+Object.keys(strings)
+  .sort()
```

Comment:
Why the sorting and mapping? Why not just `.forEach`?

Reply:
I added the sort there to have all translations sorted, I thought this could help to review the changes in git